src/BLSApkRegistryStorage.sol

```diff
@@ -29,12 +29,15 @@ abstract contract BLSApkRegistryStorage is Initializable, IBLSApkRegistry {
     /// @notice maps quorumNumber => current aggregate pubkey of quorum
     mapping(uint8 => BN254.G1Point) public currentApk;
 
+    /// @notice maps operator address to pubkey history
+    mapping(address => PubkeyCheckpoint[]) public operatorPubkeyHistory;
```

Comment:
might be worth noting (maybe an @ dev tag?) that this excludes the _current_ key? feels like something that might be easy to misinterpret, at least?